src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -122,6 +130,14 @@ export class ClrDatagridColumn<T = any> extends DatagridFilterRegistrar<T, Datag
     this._sortSubscription.unsubscribe();
   }
 
+  /*
+     * What type is this column?  This defaults to STRING, but can also be
+     * set to NUMBER.  Unsupported types default to STRING. Users can set it
+     * via the [clrDgColType] input by setting it to 'string' or 'number'.
+     */
+
+  @Input('clrDgColType') colType: 'string' | 'number' = 'string';
```

Comment:
This is fine for now, thinking we'd want to setup an enum to track these types better in the future. Maybe make a comment todo, but fine for now.